packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -482,6 +482,33 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+   <parameter>
+      <name>aggregation: brick x Dirichlet</name>
+      <type>bool</type>
+      <default>false</default>
+      <description>Asserts that Dirichlet conditions are applied in
+        the x-direction and does not aggregate these dofs.</description>
```

Comment:
Perhaps instead "and Dirichlet DOFs are not aggregated"

Reply:
got it